reference-adaptations/TFN%20Cold%20War/composition.sqe

```diff
@@ -1,19 +1,19 @@
-version=54;
-center[]={2671.946,5,6095.9458};
+version=55;
+center[]={2463.002,5,1603.022};
 class items
 {
 	items=1;
 	class Item0
 	{
 		dataType="Layer";
-		name="TFN Cold War";
+		name="TFN Cold War_1";
```

Comment:
This was probably a mistake by copying the composition folder in EDEN, thus changing the composition folder name to `TFN Cold War_1`

Reply:
Yeah, not a big deal though. It may be able to be fixed must by editing the .sqm